src/ListItem/ListItem.js

```diff
@@ -22,6 +22,7 @@ export const StyledListItem = styled.li`
   line-height: ${props => blockSizes[props.size]};
   color: ${({ theme }) => theme.materialText};
   pointer-events: ${({ isDisabled }) => (isDisabled ? 'none' : 'auto')};
+  font-weight: ${props => props.primary && 'bold'};
```

Comment:
This will evaluate to `font-weight: false` if primary is set to false. Maybe let's do something like this:
```
font-weight: ${({ primary }) => primary ? 'bold' : 'normal' };
```

Reply:
Nice! Solved @arturbien! 🎉